src/providers/services/categories/categories.service.ts

```diff
@@ -0,0 +1,15 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { environment } from 'src/environments/environment.prod';
+
+@Injectable()
+export class CategoriesService {
+
+  constructor(
+    private httpClient: HttpClient
+  ) {}
+
+  getParentCategoryList (skip: number, limit:number): any {
+    return this.httpClient.get(environment.baseUrl + 'api/category/list/parent?skip='+skip.toString() + "&limit="+ limit.toString())
```

Comment:
endPoint should be imported from environment file and pass queries in object as get method has input params.